src/app/servicos/servicos.service.ts

```diff
@@ -1,75 +1,36 @@
-import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { NGXLogger } from 'ngx-logger';
 import { Observable, of } from 'rxjs';
 import { catchError, tap } from 'rxjs/operators';
 import { environment } from 'src/environments/environment';
-import { Prestador } from '../interfaces/prestador';
 import { ServicoDetalhado } from '../interfaces/servico-detalhado';
-import { LocalStorageService } from './local-storage.service';
-
+import { SERVICOS } from '../util/tipo-servico';
+import {Servico} from '../interfaces/servico';
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6332072)